modules/ROOT/partials/konflux-key-features.adoc

```diff
@@ -1,27 +1,46 @@
+{ProductName} simplifies your DevOps processes through:
 
-* *Customizable Compliance*
+* *Automated Pipelines*: {ProductName} provides default pipeline configurations for building, testing, and deploying production software. You can trigger these pipelines via Git events, such as when team members accept push or pull requests. {ProductName} posts the results back to your source control management system.
 
-+
-Achieve the highest security standards and customize your development process to meet specific project requirements. {ProductName}'s adaptable platform ensures compliance without sacrificing flexibility.
+* *Integration with GitHub and GitLab*: {ProductName} allows you to integrate with Tekton. This integration enables you to run Tekton PipelineRuns based on GitHub or GitLab events, which you can define in your repository's `.tekton` folder. We will add more source control management providers in the future.
 
-* *Efficient Deployment*
+* *Red Hat Subscription Support*: {ProductName} supports activation keys, allowing you to access Red Hat software using tools like `buildah` and `prefetch-dependencies`, which simplifies dependency management in your software supply chain.
 
-+
-Build once, deploy anywhere. {ProductName} optimizes your release process, allowing for multiple deployments without redundancy and significantly reducing time-to-market.
+=== Collaboration
+
+{ProductName} encourages you to collaborate with its open-source community through resources such as its link:https://github.com/konflux-ci/community[community] and link:https://github.com/konflux-ci[overview pages] on GitHub.
+
+=== Customization and Flexibility
+
+{ProductName} allows you to customize your project to meet your specific requirements, maintain high security standards, and adapt to various enterprise needs. {ProductName}'s adaptable platform ensures compliance without sacrificing flexibility.
+
+=== Open Source
 
-* *Strict Data Governance*
+The core components of {ProductName} are distributed under the Apache 2.0 license and built on top of Kubernetes and OpenShift. You can set {ProductName} up locally or as a cloud-native platform for your development teams. {ProductName} utilizes standard command-line tools, such as Kubernetes' `kubectl` and OpenShift's `oc`, for managing resources.
+
+=== Robust Software Supply Chain Security
+
+{ProductName} offers unparalleled transparency into your software's build process, helping you address vulnerabilities before they become issues. {ProductName} emphasizes robust security through the following mechanisms:
+
+* *Hermetic Builds:* {ProductName} isolates your builds from changes to the host system to ensure consistency and produce accurate Software Bills of Materials (SBOMs) for tracking components, dependencies, and so forth.
```

Comment:
I feel like Hermetic builds and SBOMs are separate topics, since Konflux can generate SBOMs regardless of the build being hermetic or not.